uber/custom_tags.py

```diff
@@ -2,8 +2,8 @@
 
 
 @register.filter
-def datetime(dt, fmt='11:59pm EST on %A, %b %e'):
-    return ' '.join(dt.astimezone(c.EVENT_TIMEZONE).strftime(fmt).split())
+def datetime(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
+    return ' '.join(dt.astimezone(c.EVENT_TIMEZONE).strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm')
```

Comment:
We've always said ``11:59pm`` instead of ``11:59PM`` in the past, so I figured we'd stick with that.  I also think it looks much better that way.